turnera-app/src/components/FormContacto.js

```diff
@@ -54,6 +60,7 @@ const FormContacto = () => {
 			dispatch(setUserValues(values));
 		},
 	});
+	$("#confirmarEmail").bind("paste",function(){return false;});
```

Comment:
Hey @amadsaed don't you think that could be better to have this into a function that receives as a parameter the id that you want to disable pasting? 

Something like:
`const disablePaste = id => $(id).bind("paste", function(){return false;});`

then in useEffect you can call the function as for instance:

`useEffect(() => {
		try{
			disablePaste("#confirmarEmail");
		} catch(err){
			setError(err);
		}
	}, []);`

Reply:
got it !
I'll do it